### DISCUSSION TITLE
Tearing my hair out: Error occurred when executing IPAdapterApplyEncoded

### DISCUSSION THREAD

**Discussion by ehobbins** [comfyanonymous/ComfyUI#2731] [Q&A]
I am using ComfyUI through RunDiffusion via the cloud. I can't install it locally as I am on works machine. 

I am having a problem with a workflow for creating AI videos, and being new at this (as most of us are), and not very techncal at the best of time - I have no idea where to start!

When running my prompt all its OK, until it gets to the IPAdapter, to which  get the following error: 

```
Error occurred when executing IPAdapterApplyEncoded:

Error(s) in loading state_dict for Resampler:
size mismatch for proj_in.weight: copying a param with shape torch.Size([768, 1280]) from checkpoint, the shape in current model is torch.Size([768, 1664]).

File "/opt/rd/ComfyUI/execution.py", line 154, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "/opt/rd/ComfyUI/execution.py", line 84, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "/opt/rd/ComfyUI/execution.py", line 77, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "/opt/rd/ComfyUI/custom_nodes/ComfyUI_IPAdapter_plus/IPAdapterPlus.py", line 653, in apply_ipadapter
self.ipadapter = IPAdapter(
File "/opt/rd/ComfyUI/custom_nodes/ComfyUI_IPAdapter_plus/IPAdapterPlus.py", line 279, in __init__
self.image_proj_model.load_state_dict(ipadapter_model["image_proj"])
File "/opt/rd/miniconda3/envs/CMFY/lib/python3.10/site-packages/torch/nn/modules/module.py", line 2041, in load_state_dict
raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
```

The checkpoint i am using is photon_v1.safetensors
The yaml is photon_v1.safetensors.yaml
The VAE vae-ft-mse-840000-ema-pruned.safetensors
The IP Adapters IPAdapterPlus.py

Can anyone assist - i have pretty much tried everything. I am new at this so if my files are wrong please let me know. I am using a workflow for video by JBOOGX

**Comment by laksjdjf** [comment]
you might use wrong clip_vision.

https://github.com/cubiq/ComfyUI_IPAdapter_plus?tab=readme-ov-file#installation
